config/routes.ts

```diff
@@ -448,10 +448,6 @@
       },
     ],
   },
-  {
-    path: '/',
-    redirect: '/welcome',
```

Comment:
The `access` key is now obligatory, so it didn't work, I have no idea how to delay the check for `dashboardPermission`